src/app/features/bitcoin-choose-fee/bitcoin-choose-fee.tsx

```diff
@@ -44,7 +50,18 @@ export function BitcoinChooseFee({
   const nativeSegwitSigner = useCurrentAccountNativeSegwitIndexZeroSigner();
   const btcBalance = useNativeSegwitBalance(nativeSegwitSigner.address);
   const hasAmount = amount.amount.isGreaterThan(0);
-  const [customFeeInitialValue, setCustomFeeInitialValue] = useState(recommendedFeeRate);
+
+  const savedRateDetails: saveRateProps = JSON.parse(
+    localStorage.getItem(nativeSegwitSigner.address.toString()) || '{}'
+  );
```

Comment:
Please use redux persist for these values, so we don't rely on localstorage directly. Localstorage is a syncronous api and this runs on every render. 

Instead, let's add some state to the `settings` slice, persisting whether the user has enabled this setting, and what the value is.